docs/commands.rst

```diff
@@ -0,0 +1,73 @@
+Management Commands
+===================
+
+
+Use Case
+--------
+These management commands are designed for use when there is a need to programmatically create
+or update Django users, groups, and permissions, without needing to hardcode these in migrations or
+application code.
+
+The commands are designed to work idempotently - that is, running them multiple times
+with the same inputs should always result in the same outcomes, and without additional side-effects if they
+are run more than once.
+
+
+Put Group
+---------
+The ``put_group`` command is used to ensure that a specific group exists, and that it has a specific set of permissions.
+
+The following example will create a group named ``mygroup`` if it does not exist (and will do nothing if it does exist):
+
+.. code-block:: bash
+
+    $ python manage.py put_group mygroup
+
+
+The following example will create or update the group named ``privileged_group`` and assign to it the permissions
+for adding and deleting instances of model "Thing" in "myapp":
+
+.. code-block:: bash
+
+    $ python manage.py put_group privileged_group -p myapp:Thing:add_thing myapp:Thing:delete_thing
```

Comment:
Is there any mechanism for getting the available groups?  Seems useful for debugging.